api/organisations/tests/test_views.py

```diff
@@ -42,6 +37,14 @@
 User = get_user_model()
 
 
+class MockSubscriptionMetadata:
```

Comment:
I don't think we need this class do we? We can just use the actual `ChargebeeObjMetadata` class in the tests.

Reply:
Deleted.